libs/core/src/schemas/entities.schemas.ts

```diff
@@ -243,6 +243,16 @@ export const CommunityStake = z.object({
   updated_at: z.date().optional(),
 });
 
+export const StakeTransaction = z.object({
```

Comment:
is this an entity or a projection? guessing a projection of transaction events...

Reply:
I think it is an entity for now, but will need to be made into a projection when we convert from the command to the policy but let me know if my understanding is not correct.

Reply:
If we are planning to invoke commands on "StakeTransaction" aggregates, it's an entity... otherwise it's a projection (materialized view) of stake transactions.